src/classes/thumbnail.py

```diff
@@ -153,7 +153,7 @@ def do_GET(self):
         if not only_path:
             self.send_header('Content-type', 'image/png')
         else:
-            self.send_header('Content-type', 'text/html')
+            self.send_header('Content-type', 'utf-8')
```

Comment:
The actually-correct header:
```suggestion
            self.send_header('Content-type', 'text/html; charset=UTF-8')
```
Ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type